worker/src/main/scala/com/lucidchart/piezo/TriggerHistoryModel.scala

```diff
@@ -7,12 +7,13 @@ import org.slf4j.LoggerFactory
 import java.util.{Date, Properties}
 
 case class TriggerRecord(
-  name:String,
-  group:String,
-  scheduled_start:Date,
-  actual_start:Date,
-  finish:Date,
-  misfire:Int
+  name: String,
+  group: String,
+  scheduled_start: Date,
+  actual_start: Date,
+  finish: Date,
+  misfire: Int,
+  fire_instance_id: String
```

Comment:
I originally missed that triggers were stored in the job_history table, so I exposed this member as a way to be able to look up jobs in job_history from information in trigger_history. It is no longer needed, but it is part of the table, and I don't see any harm in exposing it.